packages/wdio-cli/src/constants.js

```diff
@@ -100,6 +101,7 @@ export const QUESTIONNAIRE = [{
     choices: [
         'On my local machine',
         'In the cloud using Sauce Labs',
+        'In the cloud using LambdaTest',
```

Comment:
Given that the only question specific to LambdaTest is username and key, we can move this to the section:
```
In the cloud using Browserstack, LambdaTest or Testingbot or a different service
```

Reply:
Done